src/components/Register/AccountDetails/SaveAccount/SaveAccount.scss

```diff
@@ -1,14 +1,19 @@
 .saveAccount {
-  display: flex;
-  align-items: flex-end;
-  flex-wrap: nowrap;
-
   .label {
     flex: 1 1 auto;
   }
+}
 
+.saveButtons {
   .button {
-    flex: 0 0 auto;
-    margin: 0 0 16px 16px;
+    width: 49%;
+    margin: 0 0 16px 0px;
+  }
+
+  #buttonNew {
+    float: left;
+  }
+  #buttonAdd {
+    float: right;
```

Comment:
We should stick with flexbox instead of using floats.  ([This](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) is a great resource if you've never used it.)  I haven't tested this, but I think this would work:

```js
.saveButtons {
  display: flex;
  align-items: center;
  align-content: stretch;
  flex-wrap: nowrap;

  .button {
    flex: 1 1 auto;

    &:first-child {
      margin-right: 8px;
    }

    &:last-child {
      margin-right: 8px;
    }
  }
}
```

Reply:
I'm not familiar with flexbox, I'm going to read that, thanks.